_config.yml

```diff
@@ -17,13 +17,10 @@ plugins:
 exclude:
   - Dockerfile
   - fetch-downloads.py
-  - Gemfile
-  - Gemfile.lock
   - nginx.default.conf
   - requirements.base
   - requirements.txt
   - thirdparty
-  - vendor
```

Comment:
these excludes no longer necessary or something? Why removed?

Reply:
https://jekyllrb.com/docs/upgrading/3-to-4/#exclusion-changes